---
title: "About Me"
author: "Meg Hemmerlein"
output:
  html_document:
    toc: yes
  word_document:
    toc: yes
  pdf_document:
    toc: yes
bibliography: references.bib
---

# Who I am and where I came from

I grew up in Contoocook, New Hampshire and played lots of sports while growing up, including soccer, alpine skiing, and lacrosse. I have always enjoyed the outdoors and therefore gravitated to becoming a scientist. The satisfaction of discovering something outweighed my interest in English literature or history.

I went to a small liberal arts college called [Wheaton College](https://wheatoncollege.edu/) in Norton, Massachusetts. I majored in biochemistry at Wheaton and also played 4 years of varsity lacrosse. During my junior year I spent a semester abroad in Christchurch, New Zealand and attended the [University of Canterbury](https://www.canterbury.ac.nz/).

After graduating Wheaton, I worked in biotechnology at a company called [Adimab](https://www.adimab.com/). I worked in the antibody engineering department. My job was to work with outside pharmaceutical companies to discovery and engineer antibodies against therapeutic targets (such as a protein related to cancer). After working with many PhD scientists over my 4 years at Adimab, I decided I wanted to advance my career as a scientist and pursue a PhD. 

I joined the Cell and Molecular Biology program at CSU in the fall of 2022. After 3 rotations I decided to join the Wilsterman lab and ask questions about the underlying molecular mechanisms driving high elevation reproductive adaptation in deer mice. I decided to take Computational Approaches to Molecular Ecology to continue adding to my foundational computational knowledge.

When I am not working I love getting out and being active. My top four things
to do are:

1. Skiing! (Alpine, Backcountry, Tele, Cross-country (in that order))
2. Mountain biking (I started coaching the SendTown bike club in Fort Collins, super fun!)
3. Rock climbing (Slab climbing outside is my favorite, dabbling in crack climbing)
4. Fly fishing 

Here is a picture of me skiing in Cameron Pass last winter.

```{r me_pic, echo=FALSE, out.width="500px"}
knitr::include_graphics("/Users/MegHemmerlein/Library/CloudStorage/OneDrive-Colostate/Spring 2024/BZ562/con-gen-csu-mjhemmerlein/images/skiing.jpg", auto_pdf = TRUE)
```


# Research Interests

I am interested in understanding molecular mechanisms underlying reproductive adaptation to high elevation in mammals.

## Influential papers

I submitted a GRFP application this fall and took inspiration from @singh2022importance. Their work described the importance of understanding alternative splicing as a mechanism for adaptive evolution, and helped me to develop my own ideas. In my scientific statement, I hypothesized that evolutionary adaptations to environmental hypoxia involved broad-scale chances in alternative splicing.

Although I study reproductive adaptation to high elevation in deer mice, I am interested in human literature describing the same adaptive phenotypes (fetal growth restriction). Lorna Moore has many influential papers, such as @moore2003fetal, which outline human adaptation to high elevation on the Tibetan and Andean plateau. 

## The mathematics behind my research

I use a package called Dream from "variancePartition" to analyze differential gene expression.  Dream uses a linear mixed model to increase power and decrease false positives. I also apply a fixed effect to my model to account for the fact that when sequencing placentas from multiple pups, the pups from the same moms will likely have more similar transcriptomic profiles to each other than the other samples. 


$$
Genecount + Population*O_2 + (1|Mom)
$$

I will additionally analyze my bulk RNA sequencing data using a weighted gene co-expression network analysis (WGCNA). Correlation networks are based on correlations between quantitative measurements and can be described by an $n$ x $m$ matrix as shown below. 

$$
X = [x_{ij}] = (x_1, x_2, ..., x_n)
$$

## My computing experience
I have experience coding in Linux and running bash scripts. This a script to align sequencing data to the deer mouse genome using STAR. The Wilsterman lab has a server where I do most of my computation work. 
```c
#!/usr/bin/env bash
 
#SBATCH --job-name=STAR-align_2Pass
#SBATCH --nodes=1
#SBATCH --ntasks=18
#SBATCH --time=unlimited
#SBATCH --output=/home/mjhemm/projects/LeafCutter/03_output/logfiles/log_STAR-align_2Pass_%J.txt

ls *.fastq.gz | while read file; do
        name=$(echo "${file}" | cut -d "_" -f 1-5)
        STAR --runThreadN 18 \
        --genomeDir /home/mjhemm/projects/LeafCutter/01_input/STAR_genome \
        --readFilesCommand gunzip -c \
        --readFilesIn ${name}_R1_001.fastq.gz ${name}_R2_001.fastq.gz \
        --sjdbFileChrStartEnd ${name}_STAR_SJ.out.tab \
        --outFileNamePrefix "${name}"_STAR_2Pass_ \
        --outSAMtype BAM SortedByCoordinate
done
```

I also have experience coding in R. I've been working on RNAseq analyses, GO enrichments and am just starting to anlayze Single-nuclei sequencing data. 

```c
##Load Data
pema_sm <- Read10X_h5(filename = "raw_feature_bc_matrix.h5")
str(pema_sm)
cts <- pema_sm
 
##Initialize the Seurat object with the raw (non-normalized data).
# Keep all features expressed in at least 3 cells (min.cells = 3)
# Keep all cells with at least 200 features/genes (min.features = 200)
pema_seu <- CreateSeuratObject(counts = cts, project = 'Pema_GEX', min.cells = 3, min.features = 200)
pema_seu

# 3. QC ===================================================
## perc_mt -----------------------
pema_seu[['percent_mt']] <- PercentageFeatureSet(pema_seu, pattern = '^MT-')
View(pema_seu@meta.data)
VlnPlot(pema_seu, features = c("nCount_RNA", "nFeature_RNA", "percent_mt"), ncol = 3)
FeatureScatter(pema_seu, feature1 = "nCount_RNA", feature2 = "nFeature_RNA") +
  geom_smooth(method = 'lm')
```

## What I hope to get out of this class

* I hope to learn how conduct my projects in a more reproducible way.
* I hope to gain exposure to ecological questions and the computational tools used to answer them.
* I hope to better understand which computational packages to implement for unique questions and projects.

# Evaluating some R code

Scatter plot of Petal length vs Sepal length from "Iris" data set.

```{r}
library(ggplot2)

ggplot(iris, aes(x = Sepal.Length, y = Petal.Length)) +
  geom_point() +
  labs(title = "Scatter Plot of Petal Length vs Sepal Length",
       x = "Sepal Length", y = "Petal Length")
```


# Citations

